pkg/instance/instance_test.go

```diff
@@ -149,8 +149,10 @@ func TestRunnerCmd(t *testing.T) {
 	flagArch := flags.String("arch", "", "target architecture")
 	flagPool := flags.Int("pool", 0, "index of pool that started VM")
 	flagVM := flags.Int("vm", 0, "index of VM that started the Runner")
+	flagNoCollide := flags.Bool("no_collide", false, "collide syscalls to provoke data races")
```

Comment:
It's always better to have positive flags ("something" rather than "no-something") for command line and normal bool variables. Otherwise there can be too many negations like "don't disable no-collide" and it may be hard to mis-interpret the meaning (contrast with "don't enable collide").

But we already have these flags:
https://github.com/google/syzkaller/blob/master/pkg/ipc/ipcconfig/ipcconfig.go#L16-L17

So you can just pass -threaded=0 to syz-runner.

Reply:
I didn't realise I can use the flags in `ipc/ipcconfig.go` and pass them to `syz-runner` (but makes sense given flags are global in Go). Thanks, should'v tried that option first!